src/interpreter/ByteCode.h

```diff
@@ -399,52 +400,25 @@ class Load32 : public ByteCode {
         DUMP_BYTECODE_OFFSET(dstOffset);
     }
 
-    virtual size_t byteCodeSize()
+    virtual void byteCodeSize()
```

Comment:
Return value should not be changed to `void`

Reply:
Thank you!
I'm working on it.